src/gui.js

```diff
@@ -468,6 +468,7 @@ IDE_Morph.prototype.openIn = function (world) {
 
         if (location.hash.substr(0, 6) === '#open:') {
             hash = location.hash.substr(6);
+            console.log(hash);
```

Comment:
looks good to me, Michael! Can you perhaps take out this `console.log` before I pull it? Tnx!

Reply:
oops! I forgot to push a couple more changes.